src/main/kotlin/fi/metatavu/muisti/realtime/mqtt/MqttConnection.kt

```diff
@@ -30,7 +30,7 @@ class MqttConnection {
             try {
                 synchronized (this) {
                     val serverURI = "tcp://${settings.serverUrl}"
-                    val client = MqttClient(serverURI, PUBLISHER_ID)
+                    val client = MqttClient(serverURI, settings.publisherId, MqttDefaultFilePersistence())
```

Comment:
what this setting does?

Reply:
publisherId setting is needed for tests, otherwise ID is generated only once in this class and 2 clients with same ID will be used and connection will fail. 
MqttDefaultFilePersistence() is not needed, it is default setting, removed it